httpserver/handlers.go

```diff
@@ -119,7 +119,18 @@ func (server *Server) verify(ctx context.Context, w http.ResponseWriter, r *http
 				verifyParameters := executor.VerifyParameters{
 					Subject: resolvedSubjectReference,
 				}
-
+				if server.GetExecutor().PolicyEnforcer == nil {
```

Comment:
as we discussed offline, we could define a method for checking everything including policy, verifier and stores.